internal/server/network/driver_bridge.go

```diff
@@ -1509,6 +1509,29 @@ func (n *bridge) Stop() error {
 		return err
 	}
 
+	// Get a list of interfaces
+	ifaces, err := net.Interfaces()
+	if err != nil {
+		return err
+	}
+
+	// Cleanup any existing tunnel and dummy devices.
+	for _, iface := range ifaces {
+		l, err := ip.LinkFromName(iface.Name)
+		if err != nil {
+			return err
+		}
+
+		if l.Master != n.name || l.Kind != "vxlan" && l.Kind != "gretap" && l.Kind != "dummy" {
```

Comment:
Would be easier to read if using `slices.Contains` for the different kinds I think.

Reply:
Done!